### DISCUSSION TITLE
invalid reference format

### DISCUSSION THREAD

**Discussion by Kramatur** [doccano/doccano#1145] [Q&A]
I am facing a issue saying "invalid reference format" while running the same command as instructed. For your reference I am writing the which I have used -

 docker container create --name doccano \ -e "ADMIN_USERNAME=admin" \ -e "ADMIN_EMAIL=admin@example.com" \ -e "ADMIN_PASSWORD=password" \ -p 8000:8000 doccano/doccano

Kindly help.

**Comment by hnal** [comment]
Hello Kramatur, 

I removed the slashes, it worked for me!
